nautiluszim/scraper.py

```diff
@@ -197,7 +195,7 @@ def run(self):
                 build_dir=self.build_dir,
                 fpath=self.output_dir / self.fname,
                 main_page="home.html",
-                favicon="favicon.png",
+                illustration="favicon.png",
```

Comment:
The argument has changed to illustration as favicon has been replaced by illustration in scraperlib 3.0.0